dm/dm/proto/dmmaster.proto

```diff
@@ -108,6 +108,17 @@ service Master {
   rpc TransferSource(TransferSourceRequest) returns(TransferSourceResponse) {}
 
   rpc OperateRelay(OperateRelayRequest) returns(OperateRelayResponse) {}
+
+   // StartValidation starts validation
+  rpc StartValidation(StartValidationRequest) returns(StartValidationResponse) {}
+  // StopValidation stop validation
+  rpc StopValidation(StopValidationRequest) returns(StopValidationResponse) {}
+
+  rpc GetValidationStatus(GetValidationStatusRequest) returns(GetValidationStatusResponse) {}
+
+  rpc GetValidationError(GetValidationErrorRequest) returns(GetValidationErrorResponse) {}
```

Comment:
just a question: if an error hanppend during validation, will the validation process be interrupted? if not, we should take care of the body size of response, it can be  larger than our imagine (maybe add pagination to solve this)

btw maybe we can merge `GetValidationStatus` to `GetValidationStatus` if so user only need one cmd to check the validation work status

Reply:
> if an error hanppend during validation, will the validation process be interrupted? if not

No, the validator will keep running until the `error-rows` reaches the `max-error-count`. It's probably necessary if the `max-error-count` is too large, but currently the number of errors won't grow infinitely. 

> btw maybe we can merge GetValidationStatus to GetValidationStatus if so user only need one cmd to check the validation work status

What do you mean about "merge `GetValidationStatus` to `GetValidationStatus`"😂?  Do you expect to merge the `query error` command to `query status` command? They are actually doing different things. The `query error` shows which rows are reported erroneous while `query status` shows the validation tasks' statuses (including whether they are still running, some details about the sources, etc.)